src/algebra/big_operators/basic.lean

```diff
@@ -57,6 +57,9 @@ protected def prod [comm_monoid β] (s : finset α) (f : α → β) : β := (s.1
   (⟨s, hs⟩ : finset α).prod f = (s.map f).prod :=
 rfl
 
+@[simp, to_additive] lemma prod_val [comm_monoid α] (s : finset α) : s.1.prod = s.prod id :=
```

Comment:
It's not clear to me that this should be simp; it sends `simp [finset.prod]` into a nasty infinite loop